libr/core/cconfig.c

```diff
@@ -970,7 +970,13 @@ static int cb_dirsrc(void *user, void *data) {
 }
 
 static int cb_cfgsanbox(void *user, void *data) {
+	RCore *core = (RCore *)user;
 	RConfigNode *node = (RConfigNode*) data;
+	if (node->i_value) {
+		const char *root = r_config_get (core->config, "http.root");
+		const char *homeroot = r_config_get (core->config, "http.homeroot");
+		r_sandbox_set_wwwpath (root, homeroot);
```

Comment:
i dont think this function name is correct. because the sandbox have nothing to do with the wwwpath and the global vars are evil, we have RCore and RConfig for a reason

Reply:
#10564